tests/TestHelpers/OcmHelper.php

```diff
@@ -80,6 +80,7 @@ public static function createInvitation(
 	 * @param string $user
 	 * @param string $password
 	 * @param string $token
+	 * @param string $providerDomain ocis-server|federation-ocis-server
```

Comment:
the server name will be different for different environment setup
```suggestion
	 * @param string $providerDomain
```